.github/workflows/docs.yml

```diff
@@ -0,0 +1,38 @@
+name: GitHub CI doc
+
+on: [push, pull_request]
+
+jobs:
+  build:
+  
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8]
+  
+    steps:
+    # https://github.com/marketplace/actions/setup-miniconda
+    - uses: actions/checkout@v2
+    - name: Install Miniconda and environment
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        activate-environment: buildh
+        environment-file: binder/environment.yml
+        python-version: ${{ matrix.python-version }}
+        channels: conda-forge,defaults
+        auto-activate-base: false
+        mamba-version: "*"
```

Comment:
GitHub CI has a time limit, per job or step, I don't know exactly. 
Conda is sometimes slow at solving (and then downloading) environment dependencies. The job failed a couple of time because of this.
[Mamba](https://github.com/mamba-org/mamba) is a much quicker implementation to solve conda dependencies.
Binder now uses mamba as default (on top of conda) and it's also possible to use mamba in [ReadTheDocs](https://docs.readthedocs.io/en/stable/guides/feature-flags.html#available-flags)

Reply:
ok thanks!